app/models/submission/content.rb

```diff
@@ -17,6 +17,13 @@ def update(step, step_content)
       merge(step => current_step_content.merge(step_content))
   end
 
+  def clear(step)
+    raise Submission::InvalidStep, "No step #{step}" unless submission.steps.include?(step.to_s)
+    submission.content = submission.
+      read_attribute(:content).
```

Comment:
No, it is here because `Submission#content` returns a `Submission::Content` instance and I want a 'raw' hash. There should be no `nil` here as a default is set before submission is created.